test/specs/phone_number.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:faker/faker.dart';
+import 'package:test/test.dart';
+
+void main() {
+  var faker = Faker();
+
+  group('person', () {
+    test('should be able to generate phone number', () {
+      for (var i = 0; i < 20; i++) {
+        expect(
+          faker.person.name(),
```

Comment:
Shouldn't be testing `faker.person.name()`?

Reply:
... indeed